lib/trailblazer/rails/railtie.rb

```diff
@@ -10,8 +10,14 @@ def self.load_concepts(app)
       # Loader.new.(insert: [ModelFile, before: Loader::AddConceptFiles]) { |file| require_dependency("#{app.root}/#{file}") }
       load_for(app)
 
-      ::Rails.application.railties.find_all { |tie| tie.is_a?(::Rails::Engine) }.each do |engine|
-        load_for(engine)
+      engines.each { |engine| load_for(engine) }
+    end
+
+    def self.engines
+      if Gem::Version.new(::Rails.version) >= Gem::Version.new("4.1")
+        ::Rails.application.railties.find_all { |tie| tie.is_a?(::Rails::Engine) }
+      else
+        ::Rails.application.railties.engines
```

Comment:
@rafaelfranca after [looking at the source for `#engines`](https://github.com/rails/rails/blob/4-0-stable/railties/lib/rails/engine/railties.rb#L12) that you linked to in [your previous comment](https://github.com/trailblazer/trailblazer-rails/pull/41#discussion_r94817912), I wonder we should just unconditionally use `::Rails::Engine.subclasses.map(&:instance)` since that's internally what all versions of Rails seem to do.